README.md

```diff
@@ -4,9 +4,8 @@
 [![Download](https://api.bintray.com/packages/avast/maven/rabbitmq-scala-client/images/download.svg) ](https://bintray.com/avast/maven/rabbitmq-scala-client/_latestVersion)
 
 
-This client is lightweight wrapper over standard [RabbitMQ java client](https://www.rabbitmq.com/java-client.html).
-It's API may be difficult to use for inexperienced RabbitMQ users. Goal of this library is to simplify basic use cases and shadow the programmer
-from the underlying client.
+This client is Scala-first wrapper over standard [RabbitMQ java client](https://www.rabbitmq.com/java-client.html). Goal of this library is
+to simplify basic use cases - to provide FP-oriented API for programmers and to shadow the programmer from an underlying client.
```

Comment:
Maybe write down a bullet list of the features provided by this library?

Reply:
Good idea.